src/DiskIO/DiskThreads/aiops_win32.cc

```diff
@@ -981,7 +981,7 @@ squidaio_poll_queues(void)
         done_queue.tailp = &done_queue.head;
 
         if (!ReleaseMutex(done_queue.mutex)) {
-            /* unexpected error */
+            assert(false); /* something is seriously broken here */
```

Comment:
Please undo this controversial out-of-scope change. If you insist on it, then more work will need to be done in this PR to make this kind of change acceptable.

Reply:
Reverting and agreeing on the out of scope, but why is it controversial?
If it's unexpected, something's broken with the state here. It might be fine to just ignore but then the comment is at leasta misleading

Reply:
> why is it controversial?

This change can be considered controversial for several reasons:

* The change modifies only one out of two identical "unexpected error" code snippets (without explaining the special treatment).
* The change kills the helper in a way that differs from existing helper killing code (without explaining the special treatment).
* A failed assertion implies a bug, but the context implies that this assertion can fail in a bug-free code.

> the comment is at leasta misleading

I disagree, but it does not really matter here.